octobot_evaluators/util/evaluation_util.py

```diff
@@ -37,3 +37,42 @@ def get_shortest_time_frame(ideal_time_frame, preferred_available_time_frames, o
         return time_frame_manager.sort_time_frames(preferred_available_time_frames)[0]
     else:
         return time_frame_manager.sort_time_frames(others)[0]
+
+
+def local_trading_context(evaluator, symbol, time_frame, trigger_cache_timestamp,
```

Comment:
I don't think we should do it like this. I don't understand full Context related need but could we use a ContextId or a channel to obtain the same result?

Reply:
Contexts are used a a proxy to access any necessary data to run a script. Data are stuff like the current symbol & timeframe, local cache, the appropriate exchange manager and identifiers to know the current tentacle class as well as its calling tentacles (in case of a nested evaluation)
In my opinion, we shouldn't use "global" or "shared" contexts as they are proxies to very local data and we should avoid the possibility of creating side effects by re-using contexts from one call to another